src/data/mystery-encounters/encounters/fiery-fallout-encounter.ts

```diff
@@ -62,16 +68,24 @@ export const FieryFalloutEncounter: MysteryEncounter =
           {
             species: volcaronaSpecies,
             isBoss: false,
-            gender: Gender.MALE
+            gender: Gender.MALE,
+            tags: [ BattlerTagType.MYSTERY_ENCOUNTER_POST_SUMMON ],
+            mysteryEncounterBattleEffects: (pokemon: Pokemon) => {
+              pokemon.scene.unshiftPhase(new StatStageChangePhase(pokemon.scene, pokemon.getBattlerIndex(), true, [ Stat.SPDEF, Stat.SPD ], 2));
```

Comment:
```suggestion
              pokemon.scene.unshiftPhase(new StatStageChangePhase(pokemon.scene, pokemon.getBattlerIndex(), true, [ Stat.SPDEF, Stat.SPD ], 1));
```
Assuming that +1 instead of +2 is still intended

Reply:
good catch, asking damo to make sure